src/plot_styles.py

```diff
@@ -197,6 +196,25 @@ def __init__(self, parent):
 
         self.present()
 
+    def on_reset_button(self, _):
+        heading = "Reset to defaults?"
+        body = "Are you sure you want to reset to the default styles?"
+        dialog = Adw.MessageDialog.new(self,
+                                       heading,
+                                       body)
+        dialog.add_response("cancel", "Cancel")
+        dialog.add_response("reset", "Reset")
+        dialog.set_close_response("cancel")
+        dialog.set_default_response("delete")
+        dialog.set_response_appearance("reset",
+                                       Adw.ResponseAppearance.DESTRUCTIVE)
+        dialog.connect("response", self.on_reset_button_press, self)
```

Comment:
I'd remove the self argument as this is a class function

Reply:
Yes, that was unnecessary, probably had it there out of habit. Even led to a unused argument in the relevant function.